Assets/UI/Popups/techciviccompletedpopup.lua

```diff
@@ -1,423 +1,436 @@
--- ===========================================================================
```

Comment:
The changes on that file are too much for just 26 insertions and 13 deletions in the update. Can you, for this file here, just replace the modified parts and not change tabs to spaces in the whole file 😉